pandas/tests/sparse/test_series.py

```diff
@@ -87,6 +87,12 @@ def setup_method(self, method):
                                       fill_value=0)
         self.ziseries2 = SparseSeries(arr, index=index, kind='integer',
                                       fill_value=0)
+    def test_constructor_data_input(self):
+        arr = SparseSeries({1: 1})
```

Comment:
Notice how you use `{1: 1}` multiple times.  Why don't we refactor that as a variable and then use that variable instead throughout the tests.  The same goes for your `index=[0, 1, 2]`.  Set that as a variable.